primitives/consensus/common/src/import_queue/basic_queue.rs

```diff
@@ -146,16 +146,48 @@ mod worker_messages {
 	pub struct ImportJustification<B: BlockT>(pub Origin, pub B::Hash, pub NumberFor<B>, pub Justification);
 }
 
-struct BlockImportWorker<B: BlockT, Transaction> {
+/// The process of importing blocks.
+///
+/// This polls the `block_import_receiver` for new blocks to import and than awaits on importing these blocks.
+/// After each block is imported, this async function yields once to give other futures the possibility
```

Comment:
Nitpick: line length

(Not sure if we bother about this tbh?)

Reply:
CI is happy :P